docs/src/usage_custom_pages.md

```diff
@@ -33,6 +33,9 @@ want to extend it with your own examples that you frequently need. In this
 case, use a file called `<command>.patch`, it will be appended to existing
 pages.
 
+> **Note:** Because the patch file will be concatenated directly to the page
+> file, it needs to start with an empty line.
+
```

Comment:
Any way we can work around that? An intermediate reader that just outputs one empty line? There is an [`impl Read for &[u8]`](https://doc.rust-lang.org/std/io/trait.Read.html#impl-Read-2)

Reply:
If we want to fix this, we should implement a custom reader, similar to what @dmaahs2017 did. Because then we can look at the ending of the page and the beginning of the patch, and only output the empty line if necessary.

However, I'd start with the current approach for now, it's easy to understand (pure concatenation) and doesn't do any magic. We can always add more heuristics, but we can't take them away without making it a breaking change.

Reply:
I wouldn't say tt's really a breaking change, the patches feature hasn't been released yet - we could add a note in the release, but I don't feel like more effort is needed. As a user, I would even prefer if I could remove all the empty lines in my personal patches over having the compatibility.

Then, it's not magic either, in the code, it is just one more `chain`:
```rust
Box::new(page_file.chain(b"\n".as_slice()).chain(patch_file))
```

Reply:
Ok, you're right, and I guess it's very unlikely that an official page already ends with an empty line. I'll add that change!